src/test/scala/uk/gov/hmrc/test/ui/pages/Transport/BorderMeansOfTransport/BorderMeansOftransportIdentificationPage.scala

```diff
@@ -27,7 +27,7 @@ object BorderMeansOftransportIdentificationPage extends RadioPage {
 
       case "IMO ship identification number"        => "imoShipIdNumber"
       case "Name of a sea-going vessel"            => "seaGoingVessel"
-      case "Train number"                          => "trainNumber"
+      case "Train Number"                          => "trainNumber"
```

Comment:
to keep this in line with the other cases should number  not be lower case and then the test adjusted to lower case. I know it's picky .